statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -332,7 +340,49 @@ def tolerance(self):
     def tolerance(self, value):
         self.ssm.tolerance = value
 
-    def fit(self, start_params=None, transformed=True,
+    def _validate_can_fix_params(self, param_names):
+        for param_name in param_names:
+            if param_name not in self.param_names:
+                raise ValueError('Invalid parameter name passed: "%s".'
+                                 % param_name)
+
+    @contextlib.contextmanager
+    def fixed_params(self, params):
+        from collections import OrderedDict
```

Comment:
is this import expensive?  if not, should go at the top of the module

Reply:
I think you're right. For some reason, in a bunch of places we have the lazy import for `collections` in particular.